### ISSUE TITLE
quay.io tags

### ISSUE THREAD

**Issue by abretaud** [erasche/docker-galaxy-annotation#5]
Oh oh oh 🎅 
I think there's something strange with the tags of this image on [quay.io](https://quay.io/repository/erasche/docker-galaxy-annotation?tag=latest&tab=tags): the `latest` image is older than the `master` image
It means that [here](https://github.com/galaxy-genome-annotation/dockerized-gmod-deployment/blob/master/docker-compose.yml#L21) we don't get the latest image (ie with the new jbrowse tool).
Don't know if this is intended?

**Comment by hexylena** [comment]
No, not intended. There's something weird with their tags lately. (Also are you abretaud on quay as well? If so, happy to add you to repos. @bgruening is your username likewise the same? I don't want a bus-factor of 1 ;))

**Comment by bgruening** [comment]
Yes, my username is the same :)

**Comment by nathandunn** [comment]
@bgruening  / @erasche  Excited to see progress on this either way.

**Comment by abretaud** [comment]
Hej and a happy new year!
Yep abretaud is me on quay.io too
